planning/behavior_path_planner/autoware_behavior_path_planner_common/src/utils/parking_departure/geometric_parallel_parking.cpp

```diff
@@ -400,10 +400,20 @@ std::vector<PathWithLaneId> GeometricParallelParking::planOneTrial(
   const Point C_far_goal_coords = inverseTransformPoint(C_far.position, arc_end_pose);
   const Point self_point_goal_coords = inverseTransformPoint(start_pose.position, arc_end_pose);
 
-  const double alpha =
-    left_side_parking
-      ? M_PI_2 - psi + std::asin((self_point_goal_coords.y - C_far_goal_coords.y) / d_C_far_Einit)
-      : M_PI_2 + psi - std::asin((self_point_goal_coords.y - C_far_goal_coords.y) / d_C_far_Einit);
+  double alpha;
+  if (is_forward) {
+    alpha =
+      left_side_parking
+        ? M_PI_2 + psi + std::asin((self_point_goal_coords.y - C_far_goal_coords.y) / d_C_far_Einit)
+        : M_PI_2 - psi -
+            std::asin((self_point_goal_coords.y - C_far_goal_coords.y) / d_C_far_Einit);
+  } else {
+    alpha =
+      left_side_parking
+        ? M_PI_2 - psi + std::asin((self_point_goal_coords.y - C_far_goal_coords.y) / d_C_far_Einit)
+        : M_PI_2 + psi -
+            std::asin((self_point_goal_coords.y - C_far_goal_coords.y) / d_C_far_Einit);
+  }
```

Comment:
```suggestion
  const double angle_offset =
    std::asin((self_point_goal_coords.y - C_far_goal_coords.y) / d_C_far_Einit);
  const double alpha = M_PI_2 + (left_side_parking ? 1.0 : -1.0) *
                                  (is_forward ? psi + angle_offset : -psi + angle_offset);
```

is this same?
I am not sure that angle_offset is appropriate name, so please replace

Reply:
It's the same procedure, and the parameter name sounds good to me.

Reply:
I modified it.